docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -0,0 +1,395 @@
+:ref:`30 microsecond pulse <telemetry_analog_request>` to the module over the throttle line.  
+
+However, as of version 4.3.6 ArduCopter flight controllers do not seem to support this method of requesting ESC telemetry according to 
+the `ArduCopter documentation (see the note on polling for telemetry data on non-DSHOT protocols) <https://ardupilot.org/copter/docs/common-blheli32-passthru.html#esc-telemetry>`_.
+The relevant note from the ArduCopter documentation is shown below.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/arducopter_no_pwm_telem.png
+    :align: center
+    :width: 60%
+    :alt: ArduCopter Note on ESC Telemetry for non-DSHOT Protocols
+
+    ArduCopter Note on ESC Telemetry for non-DSHOT Protocols
+
+DroneCAN Telemetry
+******************
+For configuring your module, follow the module set up steps in the :ref:`tutorial for integrating a Vertiq module using DroneCAN with a PX4 flight controller <dronecan_fc_tutorial>`.
+For flight controller set up, refer to the ArduCopter documentation on setting up `DroneCAN <https://ardupilot.org/copter/docs/common-uavcan-setup-advanced.html>`_ 
+and `CAN in general <https://ardupilot.org/copter/docs/common-canbus-setup-advanced.html#common-canbus-setup-advanced>`_.
+
+After completing that set up, no additional configuration of the flight controller is required for it to receive ESC telemetry. As covered in the :ref:`ESC Telemetry <telemetry_dronecan>` 
+section of the Feature Reference Manual, Vertiq modules constantly broadcast ESC telemetry messages on the DroneCAN bus when they are connected, and PX4
+flight controllers are able to accept those messages.
+
+The only additional configuration that may be useful is to adjust the :ref:`DroneCAN telemetry frequency <telemetry_dronecan_telemetry_frequency>` on
+your Vertiq module to change how frequently telemetry updates are broadcast.
+
+Check the Status section in the DATA tab of Mission Planner to see live telemetry updates from the module. Scroll over to the esc1 entries, they should be updating
+as the module broadcasts its telemetry messages, as shown in the image below.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/arducopter_dronecan_no_spin_telemetry.png
+    :align: center
+    :width: 60%
+    :alt: DroneCAN Live Telemetry Updates in the Status Section of Mission Planner
+
+    DroneCAN Live Telemetry Updates in the Status Section of Mission Planner
+
+To see a non-zero RPM you can spin the module from Mission Planner using the Motor Test tab as shown below. 
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/mp_motor_test.png
+    :align: center
+    :width: 50%
+    :alt: Motor Test in Mission Planner
+
+    Motor Test in Mission Planner to Test Telemetry When Spinning
+
+Now if you check the telemetry updates in the Status section, the reported RPM should be non-zero as can be seen below. There are no issues with conversion 

```

Comment:
same thing, maybe point out that it's on esc1_rpm. I know you say that we're using that above, but it might be nice to point out again. Not a huge deal though